src/main/java/com/minecolonies/blockout/Pane.java

```diff
@@ -403,7 +407,7 @@ public final Window getWindow()
         return window;
     }
 
-    protected void setWindow(final Window w)
+    public void setWindow(final Window w)
```

Comment:
Oddly I needed here, I think because I was directly calling the method on a Pane 🤷‍♂️ 
```
    @Override
    protected void setWindow(final Window w)
    {
        super.setWindow(w);
        for (final Pane child : children)
        {
            child.setWindow(w);
        }
    }
```

The `child.setWindow` complained. Even though `View extends Pane`